File: C#/0001.cs
using System;

namespace SumOfTwoNumbers
{
    class Program
    {
        static void Main(string[] args)
        {
            string input = Console.ReadLine();
            string[] numbers = input.Split(' ');

            int var1 = Convert.ToInt32(numbers[0]);
            int var2 = Convert.ToInt32(numbers[1]);

            int sum = var1 + var2;
            Console.WriteLine(sum);
        }
    }
}

File: Golang/0001.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	fmt.Print()
	input, _ := reader.ReadString('\n')
	input = strings.TrimSuffix(input, "\n")

	numbers := strings.Split(input, " ")
	var1, _ := strconv.Atoi(numbers[0])
	var2, _ := strconv.Atoi(numbers[1])

	sum := var1 + var2
	fmt.Println(sum)
}